### ISSUE TITLE
pip install failing due to tensorflow

### ISSUE THREAD

**Issue by ortsaCniveK** [RasaHQ/rasa#5722]
**Rasa version**: 1.9.3

**Python version**: 3.8.2

**Operating system** (windows, osx, ...): Linux

**Issue**: `pip install rasa` fails with below traceback.  Additionally fails even with tensorflow installation due to `tensorflow-addons` requirement.

**Error (including full traceback)**:
```
$ pip install --no-cache-dir rasa
Collecting rasa
  Downloading rasa-1.9.7-py3-none-any.whl (497 kB)
     |████████████████████████████████| 497 kB 2.3 MB/s 
Collecting python-socketio<4.5,>=4.4
  Downloading python_socketio-4.4.0-py2.py3-none-any.whl (50 kB)
     |████████████████████████████████| 50 kB 5.1 MB/s 
Requirement already satisfied: setuptools>=41.0.0 in ./env/lib/python3.8/site-packages (from rasa) (44.0.0)
Collecting rocketchat_API<0.7.0,>=0.6.31
  Downloading rocketchat_API-0.6.36-py3-none-any.whl (9.5 kB)
Requirement already satisfied: numpy<2.0,>=1.16 in ./env/lib/python3.8/site-packages (from rasa) (1.18.3)
Collecting sanic-jwt<1.4.0,>=1.3.2
  Downloading sanic-jwt-1.3.2.tar.gz (18 kB)
Requirement already satisfied: absl-py<0.10,>=0.9 in ./env/lib/python3.8/site-packages (from rasa) (0.9.0)
ERROR: Could not find a version that satisfies the requirement tensorflow<2.2,>=2.1 (from rasa) (from versions: 2.2.0rc1, 2.2.0rc2, 2.2.0rc3)
ERROR: No matching distribution found for tensorflow<2.2,>=2.1 (from rasa)
```

**Command or request that led to error**:
```
pip install rasa
```

**Comment by Talhag958** [comment]
I faced the same issue
So I run command 
pip install tensorflow 
and then run command,
pip install rasa
again. this worked for me.

**Comment by sara-tagger** [comment]
Thanks for raising this issue, @dakshvar22 will get back to you about it soon✨

###### Please also check out the [docs](https://rasa.com/docs/) and the [forum](https://forum.rasa.com/) in case your issue was raised there too 🤗

**Comment by ortsaCniveK** [comment]
Confirming this is no longer an issue as of today;  I'm guessing it was an issue with the Tensorflow release, which is unrelated to anything in Rasa. Closing.

**Comment by MaveriQ** [comment]
i am getting this problem again. I have python 3.8.2 and I am using pip to install rasa

**Comment by ortsaCniveK** [comment]
Based on #5822, it seems that only python 3.6 & 3.7 are supported currently

**Comment by Arpit12gupta** [comment]
I have same issue, Please help me out...

**Comment by 3vts** [comment]
Can confirm the same error is happening rn

**Comment by ortsaCniveK** [comment]
@Arpit12gupta @3vts what python version are you currently on?

3.8 support wasn't added until the 2.0 pre-release, so what you could do for now is:

```sh
$ pip install --pre rasa
```

to install the latest pre-release version of Rasa.  Just tested with Python 3.8 and was able to install successfully.

**Comment by 3vts** [comment]
Thanks! It works now

**Comment by woelfle2004** [comment]
If you still experience problems try this guide
https://www.youtube.com/watch?v=4ewIABo0OkU

**Comment by Saurabh211997** [comment]
I found error type sparse dropout after train rasa 
Any have solution for this
